### PR TITLE
Add Indonesian translation

### PR DESCRIPTION
**Pull request by kribolista**
- [x] translate metadata
- [x] translate white-paper

### CODE REVIEW

File: i18n/id/white-paper.md [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+Pemilihan yang bias ini menyebabkan batu bata penting yang menopang menara tidak menarik sumbangan, sementara batu bata favorit menerima lebih banyak dari yang mereka butuhkan.
+Hadiah memungkinkan konsumen proyek untuk mengusulkan pembayaran bagi pengembang untuk membangun fitur tertentu, sehingga hanya memberi imbalan kepada proyek untuk melakukan hal-hal yang belum tentu demi kepentingan terbaik mereka. Dan sekali lagi, hanya memberi penghargaan kepada proyek favorit.
+
+Dalam makalah ini, kami mengusulkan tea - sistem terdesentralisasi untuk memberikan remunerasi yang adil kepada pengembang sumber terbuka berdasarkan kontribusi mereka terhadap seluruh ekosistem dan diberlakukan melalui algoritme insentif tea yang diterapkan di semua entri dalam registri tea.
+
+[Tampilan sederhana dari sistem imbalan seduhan tea.] (img/figure-1.svg)
```
Comment (mfts) [inline i18n/id/white-paper.md:31]: Please add `!` in front of the [] to indicate it's an image and no spaces between `[]()`

File: i18n/id/white-paper.md [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+
+Skalabilitas, efektivitas biaya, ESG, dan ekstensibilitas pihak ketiga adalah pertimbangan desain penting yang dapat dilayani dengan lebih baik oleh sistem bukti kepemilikan tea. Dalam proof-of-stake, operator node dan peserta jaringan mempertaruhkan nilai ekonomi dalam bentuk token asli rantai untuk meningkatkan keamanan sistem. Operator node dan peserta jaringan menerima imbalan atas keberhasilan produksi blok yang mematuhi aturan jaringan dan menyertakan informasi transaksi yang akurat. Ketidakaktifan (alias node down) atau aktivitas jahat / salah dihukum dengan menghancurkan sebagian kecil token yang dipertaruhkan melalui pemotongan.
+
+Sistem bukti kepemilikan yang didukung oleh token tea akan memungkinkan pemegang token tea untuk berkontribusi pada keamanan sistem dengan *mengambil* tea dan mendukung pengembang sumber terbuka dengan *menyeduh* tea. Kami sepenuhnya menyadari bahwa faktor ekonomi dapat mencegah beberapa pengembang untuk mengintai atau menyeduh tea; dengan demikian, mengintai dan menyeduh akan tersedia hanya dengan satu lembar daun, denominasi terkecil tea yang mewakili seperseratus juta ($ 10 ^ {{-8} $) tea.
```
Comment (mfts) [inline i18n/id/white-paper.md:234]: > ($ 10 ^ {{-8} $)

Something odd happened here. The formula is not displayed properly. Please double-check in the `white-paper.md` for reference.

File: i18n/id/white-paper.md [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+
+Tujuan tea adalah untuk mewakili nilai perangkat lunak sumber terbuka dan, dengan demikian, mendorong pertumbuhannya dengan memberdayakan para pesertanya dengan sumber daya yang mereka butuhkan untuk mengejar hasrat mereka tanpa terbebani. Sistem distribusi insentif tea perlu mempertimbangkan dengan cermat rasio seduhan setiap paket dan menyesuaikan insentif setiap paket dengan tepat. Untuk mengurangi risiko sejumlah kecil paket yang digunakan sebagai dependensi di banyak aplikasi yang mengumpulkan sebagian besar reward menyeduh, kami akan memanfaatkan penelitian yang dihasilkan oleh web3 Foundation[^ 19] untuk mekanisme reward berbasis bukti kepemilikan Polkadot. Kita bisa menyesuaikan lebih lanjut implementasi dan variabel-variabelnya berdasarkan hasil eksperimen praktis.
```
Comment (mfts) [inline i18n/id/white-paper.md:244]: > [^ 19]

There should be no space between the characters --> [^19]

File: i18n/id/white-paper.md [hunk 4]
```diff
@@ -0,0 +1,361 @@
...
...un secara progresif. Ketika sebuah paket melebihi rasio seduhan optimalnya, rasio imbalan seduhan akan menurun tajam untuk menghilangkan insentif bagi pendukung paket dan pencicip tea dari seduhan lebih lanjut paket yang sangat curam. Desain ini dapat memungkinkan paket yang kurang diseduh menjadi lebih menarik bagi pendukung paket dan pencicip tea. Hal ini juga dapat memberi insentif bagi para pengembang berpengalaman untuk membangun alternatif bagi paket yang sangat canggih, menciptakan peluang bagi komunitas tea untuk menyeimbangkan antara mendukung perangkat lunak yang sudah ada dan mempromosikan inovasi. Rasio menyeduh akan dihitung dengan menggunakan suplai sirkulasi dalam desain awalnya. Komunitas tea dapat mengubah desain ini untuk meningkatkan skalabilitas sistem lebih lanjut. Biarkan $\chi$ menjadi rasio seduhan di semua paket. Ini mewakili jumlah total token tea yang diseduh oleh manajer/pengelola paket, pengguna paket, pendukung dan dukungan paket, dan pencicip tea dibagi dengan total pasokan token tea. Mengingat banyaknya paket sumber terbuka yang tersedia saat ini dan pertumbuhan yang diharapkan, $\chi$ akan selalu menjadi nilai yang sangat kecil antara $0$$ dan $1$$.
```
Comment (mfts) [inline i18n/id/white-paper.md:246]: > $0$$ dan $1$$

too many `$` symbols. It should be only `$0$` and `$1$`.

File: i18n/id/white-paper.md (hunk 3 above)
Comment (mfts) [reply i18n/id/white-paper.md:244]: Please double-check your references. According to the white-paper.md file, this should be reference `[^22]`

**Review by mfts — CHANGES_REQUESTED** [review]
Please see my change requests inline.

**Comment by kribolista** [comment]
- [x] Please add ! in front of the [] to indicate it's an image and no spaces between []()
- [x] ($ 10 ^ {{-8} $)
- [x] There should be no space between the characters --> [^19]
- [x] too many $ symbols. It should be only `$0$` and `$1$`

**Comment by aalifax** [comment]
hi, can I get involved in this translation project?

**Review by mfts — APPROVED** [review]
Thank you for implemented the changes, @kribolista.

**Comment by mfts** [comment]
@Xercesblu3 this is ready to merge from my point of view

**Comment by mfts** [comment]
@Xercesblu3 please wait on the merge until we figured out the situation with the other Indonesian translations. I'm currently in contact with all 4 translators.

**Pull request closed without merging**